Formula/consul-template.rb

```diff
@@ -16,50 +18,49 @@ class ConsulTemplate < Formula
 
   devel do
     url "https://github.com/hashicorp/consul-template.git",
-        :tag => "v0.18.0-rc1",
-        :revision => "933192dc0b6bb642af2549848e0681670550e095"
-    version "0.18.0-rc1"
-
-    # Upstream issue "Request: Makefile target that doesn't use docker"
-    # Reported 18 Nov 2016 https://github.com/hashicorp/consul-template/issues/793
-    patch :DATA
-  end
-
-  head do
-    url "https://github.com/hashicorp/consul-template.git"
-
-    patch :DATA
+        :tag => "v0.18.0-rc2",
+        :revision => "e446ab16fb567e1ddeb88ef29406b53bdb4ee387"
+    version "0.18.0-rc2"
   end
 
   depends_on "go" => :build
 
+  # This can be removed on next stable version
   go_resource "github.com/mitchellh/gox" do
     url "https://github.com/mitchellh/gox.git",
         :revision => "c9740af9c6574448fd48eb30a71f964014c7a837"
   end
 
   # gox dependency
+  # This can be removed on next stable version
   go_resource "github.com/mitchellh/iochan" do
     url "https://github.com/mitchellh/iochan.git",
         :revision => "87b45ffd0e9581375c491fef3d32130bb15c5bd7"
   end
 
   def install
-    ENV["GOPATH"] = buildpath
-
-    dir = buildpath/"src/github.com/hashicorp/consul-template"
-    dir.install buildpath.children
-
-    Language::Go.stage_deps resources, buildpath/"src"
-    ENV.prepend_create_path "PATH", buildpath/"bin"
-    cd("src/github.com/mitchellh/gox") { system "go", "install" }
-
-    cd dir do
-      system "make", "dev"
-      system "make", "test"
+    gopath = buildpath
+    (gopath/"src/github.com/hashicorp/consul-template").install buildpath.children - [buildpath/".brew_home"]
+
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"
```

Comment:
The `buildpath/"bin"` directory only needs to be in the `PATH` if it's the stable spec and we're using `gox`, right? This can be moved into the `if build.stable?` below.